### PR TITLE
Deaccession CLI import task, refs #12020

### PR DESCRIPTION
**Pull request by sbreker**
Add new CLI task to import Deaccession records.

Default behaviour:
- deaccession row much match an existing accession number to be
imported. If match not found, an error will be displayed and the row
will not be loaded.

- deaccession row will be checked to make sure that it does not exactly
match an existing row in the database. If it does, an error will be
displayed and the row will not be loaded.

- dupe checking can be overridden so that the entire csv will be loaded
without checking for dupes using the --ignore-duplicates option.

### CODE REVIEW

File: lib/task/import/csvDeaccessionImportTask.class.php [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        'scope',
+        'accessionNumber',
+      ),
+
+      // Import logic to load deaccession
+      'rowInitLogic' => function(&$self)
+      {
+        $accessionIdentifier =  $self->rowStatusVars['accessionNumber'];
+
+        $ignoreDuplicates = ($self->status['options']['ignore-duplicates']) ? true : false;
```
Comment (mcantelon) [inline lib/task/import/csvDeaccessionImportTask.class.php]: Is this row necessary?

File: lib/task/import/csvDeaccessionImportTask.class.php [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+ *
+ * @package    symfony
+ * @subpackage task
+ * @author     Steve Breker <sbreker@artefactual.com>
+ */
+class csvDeaccessionImportTask extends csvImportBaseTask
+{
+  protected $namespace           = 'csv';
+  protected $name                = 'deaccession-import';
+  protected $briefDescription    = 'Import csv deaccession data';
```
Comment (mcantelon) [inline lib/task/import/csvDeaccessionImportTask.class.php]: Ultra-nitpick with CSV capitalization. ;)

**Comment by mcantelon** [comment]
Other than a couple of super minor nitpicks, looks good!

**Comment by sbreker** [comment]
Suggested changes made.

**Comment by mcantelon** [comment]
👍

**Pull request merged**